ssl/ssl_aead_ctx.cc

```diff
@@ -21,6 +21,7 @@
 #include <openssl/err.h>
 #include <openssl/rand.h>
 
+#include "../crypto/fipsmodule/cipher/internal.h"
```

Comment:
Don't disagree, in this instance I think the trade-off was to avoid increasing the public API surface area for the EVP_AEAD functions with the serialization/deseralization support until this an external customer requirement to do so.